plugin-server/package.json

```diff
@@ -5,10 +5,9 @@
     "types": "dist/index.d.ts",
     "main": "dist/index.js",
     "scripts": {
-        "test": "jest --runInBand --forceExit tests/**/*.test.ts",
-        "test:postgres": "jest --runInBand --forceExit tests/postgres/",
-        "test:clickhouse:1": "jest --runInBand --forceExit tests/ --testPathIgnorePatterns tests/shared/ --testPathIgnorePatterns tests/postgres/",
-        "test:clickhouse:2": "jest --runInBand --forceExit tests/shared/",
+        "test": "jest --runInBand --forceExit tests/",
+        "test:1": "jest --runInBand --forceExit tests/ --testPathIgnorePatterns tests/shared/",
+        "test:2": "jest --runInBand --forceExit tests/shared/",
```

Comment:
`shared` feels like a weird name

Reply:
I agree. Rearranged the directory structure a bit so that it's `main` and `worker` (instead of `clickhouse`, `shared`, and `worker`)